packages/blocks/src/lib/maplibre/index.svelte

```diff
@@ -102,22 +116,20 @@ onMount(() => {
     maxPitch,
     minZoom,
     maxZoom,
+    ...options,
   });
 
   context.map.set(map);
 
-  const nav = new NavigationControl({ showZoom: false });
-  map.addControl(nav, 'top-right');
-
   map.on('move', handleMove);
   map.on('resize', handleResize);
   map.on('style.load', handleCreate);
-});
 
-onDestroy(() => {
-  map?.off('move', handleMove);
-  map?.off('resize', handleResize);
-  map?.off('style.load', handleCreate);
+  return () => {
```

Comment:
I've been avoiding `onDestroy` when possible. It runs server side whereas the mount cleanup function does not.